src/parsers/manifestjson.js

```diff
@@ -99,6 +102,25 @@ export default class ManifestJSONParser extends JSONParser {
       );
       this.io = io;
       this._validate();
+
+      if (this.parsedJSON.default_locale) {
+        try {
+          // TODO should this also have the relaxed parsing? -> optimally this would use the same parser run as the check for the messages.json file.
```

Comment:
This is honestly the implementation part that I'm the most unsure of. The goal is to have the default messages available inside the eslint/stylelint rules. To do so, the messages are set on the metadata object. However the manifest.json parser runs before all other file parsers. To avoid duplicate work, optimally this wouldn't have to parse the default messages file and instead would just retrieve the parsed version from somewhere, or have it get set.